lib/web-server.js

```diff
@@ -77,6 +59,29 @@ class WebServer {
 
 
     /**
+     * Attaches error handling to the server. This captures different error types,
+     * suppresses their output to console or to the client and then they get caught
+     * by the Uncaught Exception Handler.
+     */
+    attachErrors() {
```

Comment:
We are attaching error handlers (attachErrorHandlers) or capturing errors (captureErrors) but we are not attaching errors (attachErrors).